astropy/units/deprecated.py

```diff
@@ -41,6 +47,7 @@ def _initialize_module():
 # This generates a docstring for this module that describes all of the
 # standard units defined here.
 from .utils import generate_unit_summary as _generate_unit_summary  # noqa
+from .utils import generate_prefixonly_unit_summary as _generate_prefixonly_unit_summary  # noqa
```

Comment:
oops!  Good catch - it is indeed supposed to be called below.  Looks like I did that locally but forgot to commit this file with the second line...